packages/magento-search/components/CategorySearchResult/CategorySearchResult.tsx

```diff
@@ -53,7 +52,6 @@ export function CategorySearchResult(props: CategorySearchResultProps) {
               text={breadcrumb?.category_name ?? ''}
               highlight={search}
             />
-            {' / '}
```

Comment:
You may want to still have a `/` if the breadcrumbs show a path thats deeper then 1 level. Maybe only show a `/` if `length > 1` and `index > 0` ?